README.md

```diff
@@ -104,7 +104,7 @@ export default class Tree extends Component {
 | dndType                        |     string     | String value used by [react-dnd](http://react-dnd.github.io/react-dnd/docs-overview.html) (see overview at the link) for dropTargets and dragSources types. If not set explicitly, a default value is applied by react-sortable-tree for you for its internal use. **NOTE:** Must be explicitly set and the same value used in order for correct functioning of external nodes                                                                                                                                                                                                                                                                     |
 | shouldCopyOnOutsideDrop        |  func or bool  | Return true, or a callback returning true, and dropping nodes to react-dnd drop targets outside of the tree will not remove them from the tree. Defaults to `false`. <div>`({ node: object, prevPath: number[] or string[], prevTreeIndex: number, }): bool`</div>                                                                                                                                                                                                                                                                                                                                                                                 |
-| reactVirtualizedListProps      |     object     | Custom properties to hand to the [react-sortable-tree list](https://github.com/bvaughn/react-sortable-tree/blob/master/docs/List.md#prop-types)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| reactVirtualizedListProps      |     object     | Custom properties to hand to the [react-sortable-tree list](https://github.com/frontend-collective/react-sortable-tree/blob/master/src/react-sortable-tree.js#L757)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
hi, the correct link is actually https://github.com/bvaughn/react-virtualized/blob/master/docs/List.md#prop-types

I can merge it once it is updated. 

thanks!